components/ws-manager-mk2/service/manager.go

```diff
@@ -911,11 +934,16 @@ func extractWorkspaceStatus(ws *workspacev1.Workspace) *wsmanapi.WorkspaceStatus
 		tpe = wsmanapi.WorkspaceType_REGULAR
 	}
 
-	var timeout string
+	timeout := wsm.Config.Timeouts.RegularWorkspace.String()
```

Comment:
made this change after seeing how you did it for the closed timeout below 👍 Also aligns with mk1 now